charts/ingress-nginx/values.yaml

```diff
@@ -808,6 +817,15 @@ controller:
             - ALL
         readOnlyRootFilesystem: true
       resources: {}
+      extraVolumeMounts: []
+      ## Additional volumeMounts to the AdmissionsWebhook patchWebhookJob.
+      #  - name: kube-api-access
+      #    mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      #    readOnly: true
+      extraVolumes: []
+      ## Additional volume to the AdmissionsWebhook patchWebhookJob.
+      #  - name: kube-api-access
+      #    emptyDir: {}
```

Comment:
Please put the comment before the actual property and use `# --` notation for documentation.